aicsimageio/aics_image.py

```diff
@@ -478,10 +478,7 @@ def get_physical_pixel_size(self, scene: int = 0) -> Tuple[float]:
         sizes: Tuple[float]
             Tuple of floats representing the pixel sizes for X, Y, Z, in that order.
         """
-        try:
-            return self.reader.get_physical_pixel_size(scene)
-        except AttributeError:
-            return (1.0, 1.0, 1.0)
+        return self.reader.get_physical_pixel_size(scene)
```

Comment:
Can you just do `self.get_physical_pixel_size = self.reader.get_physical_pixel_size` in the `__init__`? 🤔 I wonder if that might help de-duplicate the docstrings/still work with the doc building

Reply:
Sphinx is smart but not that smart unfortunately.